CONTRIBUTING.md

```diff
@@ -70,13 +70,24 @@ npm install checkly@0.0.0-pr.<PR-NUMBER>.<COMMIT_SHORT_SHA>
 
 ## Releasing
 
+### Releasing `checkly` (CLI)
+
 To release the project to NPM:
 
-1. Update the `version` field in [package/package.json](./package/package.json)
+1. Update the `version` field in the [CLI package.json](./packages/cli/package.json)
```

Comment:
We can use the `npm version <new_version> --workspace=packages/cli` command to automatically change the `package-lock.json` aswell.

![image](https://user-images.githubusercontent.com/5315705/235922223-367c8ac5-428b-4724-b22e-f49a6383fa8d.png)